uabot.code-snippets

```diff
@@ -0,0 +1,741 @@
+      "\t},\r",
+      "\t\"badQueriesInLanguage\": {\r",
+      "\t},\r",
+      "\t\"randomCustomData\": [\r",
+			"\t\t{\r",
+			"\t\t\t\"apiname\": \"c_isbot\",\r",
+			"\t\t\t\"values\": [\r",
+			"\t\t\t\t\"true\"\r",
+			"\t\t\t]\r",
+			"\t\t},\r",
+      "\t],\r",
+      "\t\"scenarios\": [\r",
+			"\t\t$0\r",
+      "\t]\r",
+      "}"
+    ],
+    "description": "Coveo UABot Simple Configuration"
+	},
+
+	"UABot Random Custom Data": {
+    "prefix": "uabotrandomcustomdata",
+    "scope": "plaintext,json",
+    "body": [
+      "{\r",
+      "\t\"apiname\": \"$1\",\r",
+      "\t\"values\": [\r",
+      "\t\t\"$2\"\r",
+      "\t]\r",
+      "},$0"
+    ],
+    "description": "Coveo UABot Random Custom Data"
+	},
+
+  "UABot Complete Scenario": {
+    "prefix": "uabotscenario",
+    "scope": "plaintext,json",
+    "body": [
+      "{\r",
+      "\t\"name\": \"$1\",\r",
+      "\t\"weight\": ${2:Weight for randomizing scenarios (integer)},\r",
+      "\t\"useragent\": \"$3\",\r",
+      "\t\"mobile\", false,\r",
+      "\t\"language\": \"$4\"\r",
+			"\t\"events\": [\r",
+			"\t\t$0\r",
+      "\t]\r",
+      "},"
+    ],
+    "description": "Coveo UABot Complete Scenario"
+	},
+
+  "UABot Simple Scenario": {
+    "prefix": "uabotscenario",
+    "scope": "plaintext,json",
+    "body": [
+      "{\r",
+      "\t\"name\": \"$1\",\r",
+      "\t\"weight\": ${2:Weight for randomizing scenarios (integer)},\r",
+      "\t\"events\": [\r",
+			"\t\t$0\r",
+      "\t]\r",
+      "},"
+    ],
+    "description": "Coveo UABot Simple Scenario"
+	},
+
+	"UABot Complete Search Event": {
+		"prefix": "uaboteventsearch",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"Search\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"queryText\": \"${1:Optional. The query to send. Leave \"blank\" for a random query}\",\r",
+			"\t\t\"goodQuery\": ${2:Optional. true/false},\r",
+			"\t\t\"matchLanguage\": ${3:Optional. true/false},\r",
+			"\t\t\"caseSearch\": ${4:Optional. If the query comes from a Case Creation interface. true/false},\r",
+			"\t\t\"inputTitle\": \"${5:Optional. The title of the input that triggered the search if it was a case search}\",\r",
+			"\t\t\"actionCause\": \"${6:Optional}\",\r",
+			"\t\t\"ignoreEvent\": ${7:Optional. true/false},\r",
+			"\t\t\"customData\": {\r",
+			"\t\t\t$0\r",
+			"\t\t}\r",
+			"\t}\r",
+			"},"
+		],
+		"description": "Coveo UABot Complete Search Event"
+	},
+
+	"UABot Event Custom Data": {
+		"prefix": "uaboteventcustomdata",
+		"scope": "plaintext,json",
+		"body": [
+			"\"customData\": {\r",
+			"\t$0\r",
+			"}"
+		],
+		"description": "Coveo UABot Event Custom Data"
+	},
+
+	"UABot GoodQuery Search Event": {
+		"prefix": "uaboteventsearch",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"Search\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"goodQuery\": true,\r",
+			"\t\t\"matchLanguage\": ${1:Optional. true/false}\r",
```

Comment:
would remove, this makes it overly complex and is only when you have multiple languages in the same scenario config file.